app/build.gradle

```diff
@@ -277,14 +286,20 @@ gradle.taskGraph.whenReady {
             if (!android.signingConfigs.release.storeFile.exists()) {
                 throw new IllegalStateException("To perform a release build, you must have the release repo cloned at ${releaseRepoRoot}.")
```

Comment:
This error message is out of date—it should say something like "there is no keystore file at <path>".

Reply:
Fixed